src/login2/registration.rs

```diff
@@ -0,0 +1,144 @@
+use adw::prelude::MessageDialogExtManual;
+use adw::traits::MessageDialogExt;
+use gettextrs::gettext;
+use glib::subclass::InitializingObject;
+use glib::Properties;
+use gtk::gio;
+use gtk::glib;
+use gtk::glib::clone;
+use gtk::prelude::*;
+use gtk::subclass::prelude::*;
+use gtk::CompositeTemplate;
+
+use crate::tdlib as model;
+use crate::utils::parse_formatted_text;
+use crate::utils::spawn;
+
+mod imp {
+    use super::*;
+
+    #[derive(Debug, Default, CompositeTemplate, Properties)]
+    #[template(resource = "/com/github/melix99/telegrand/ui/login-registration.ui")]
+    #[properties(wrapper_type = super::Registration)]
+    pub(crate) struct Registration {
```

Comment:
Registration is not working anymore for some time because of recent internal Telegram changes. Let's use this opportunity to remove this page and show a message that suggests to use an official client to register instead.

Reply:
At least registration still works for test accounts. But we can remove registration later.